tests/onnxruntime/testing_utils.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+import torch
+
+from optimum.onnxruntime.utils import get_device_for_provider, get_provider_for_device
+
+
+class TestProviderAndDeviceGetters(unittest.TestCase):
```

Comment:
We should rename this file `test_*.py` (https://docs.pytest.org/en/7.1.x/getting-started.html#run-multiple-tests) or adapt the onnxruntime workflows